### PR TITLE
Add AVS list to Operator details

### PR DESCRIPTION
**Pull request by vincenthongzy**
After discussing with Clem, simple list for now.

### CODE REVIEW

File: src/avs/AVSDetails.jsx [hunk 1]
```diff
@@ -105,7 +105,7 @@ export default function AVSDetails() {
           cursor: 'rounded border border-outline shadow-none',
           panel: 'p-0',
           tab: 'h-fit p-2',
-          tabList: 'rd-box w-full p-2'
+          tabList: 'rd-box w-full !overflow-x-scroll p-2'
```
Comment (vincenthongzy) [inline src/avs/AVSDetails.jsx:108]: @rubo the `rd-box`s overflow hidden caused the tab list horizontal scroll to stop working on smaller screens

Comment (rubo) [reply src/avs/AVSDetails.jsx:108]: It works now on a mobile. What's the problem?

Comment (vincenthongzy) [reply src/avs/AVSDetails.jsx:108]: It fails when there are errors. The error messages causes each tab to be longer than usual, causing tabs to be cut off.

Comment (rubo) [reply src/avs/AVSDetails.jsx:108]: Got it. I'll check but for this particular tab disabling it is ok.

**Review by Zishan-7 — APPROVED** [review]

File: src/avs/AVSDetails.jsx [hunk 2]
```diff
@@ -122,7 +122,7 @@ export default function AVSDetails() {
               </div>
               <div className="flex w-full justify-center">
                 {state.isAVSLoading && (
-                  <Skeleton className="mt-2 h-4 w-full rounded-md" />
+                  <Skeleton className="mt-2 h-4 w-full min-w-16 rounded-md" />
```
Comment (rubo) [inline src/avs/AVSDetails.jsx:125]: For short texts like "TVL", skeleton expansion is too short. So, added a min width.

**Review by rubo — APPROVED** [review]

**Pull request merged**